### ISSUE TITLE
Translation Interface Maybe？

### ISSUE THREAD

**Issue by A1oneR** [frederik-hoeft/rimworld-more-injuries#3]
It seem like there is no interface to translate the file in assemblies, it will be appreciate to see the change so that others can translate the remain text for the non-native English speakers.

**Comment by frederik-hoeft** [comment]
Good call, I'll look into it. Haven't worked with localization in RimWorld before, but as there are many other mods out there that have done it, I'm sure it should be easy enough to implement
